jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/IndexedResources.kt

```diff
@@ -10,7 +10,13 @@ package software.aws.toolkits.jetbrains.services.cloudformation
 open class IndexedResource internal constructor(val indexedProperties: Map<String, String>) {
 
     protected constructor(resource: Resource, indexProperties: List<String>)
-            : this(indexProperties.map { it to resource.getScalarProperty(it) }.toMap())
+            : this(indexProperties.map {
+        it to try {
+            resource.getScalarProperty(it)
+        } catch (e: Exception) {
+            ""
```

Comment:
I'd rather have `null` than empty string - at least that's something that the compiler forces me to handle.

Reply:
Changed to `null` and also refactored the persistence code to the `IndexedResource` class

Reply:
Did you push the code? Looks like it's still returning empty string...

Reply:
Yes, I did push two separate commits. Any more concerns?